external/netcoreapp/netcoreapp.depproj

```diff
@@ -19,7 +27,8 @@
   <ItemGroup>
     <!-- for all configurations this project provides refs for that configuration -->
     <BinPlaceConfiguration Include="$(Configuration)">
-      <RefPath>$(RefPath)</RefPath>
+      <RefPath Condition="'$(TargetsWindows)' != 'true'">$(RefPath)</RefPath>
```

Comment:
I think you should use BinPlaceRef and BinPlaceRuntime properties here instead of TargetsWindows.